tests/unit/test_cube.py

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+
+"""Unit test suite for `cr.cube.cube` module."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from cr.cube.cube import Cube
+
+
+class DescribeBaseCrunchCube(object):
+    def it_raises_a_type_error_if_not_serializable_object_provided(self):
```

Comment:
In general, exception-path tests should be paired with the "happy-path" test. So here would go:

```python
def it_provides_access_to_the_cube_response_dict_to_help(self):
    # ---fixturing---

    cube_dict = cube._cube_dict

    # ---assertions about cube_dict---

def and_it_accepts_a_JSON_format_cube_response(self):
    # ---fixturing ---

    cube_dict = cube._cube_dict

    # ---assertions about cube_dict---

def but_it_raises_on_other_cube_response_types(self):
    # ---fixturing---

    with pytest.raises(TypeError) as e:
        cube._cube_dict

    # ---assertions about e---